src/puppeteer-core.ts

```diff
@@ -67,13 +71,13 @@ class PuppeteerWorkers extends Puppeteer {
         `Unabled to create new browser: code: ${status}: message: ${text}`
       );
     }
-    // Got a 200, so response text is actually a session id
-    const sessionId = text;
+    // Got a 200, so response text is actually an AcquireResponse
+    const response: AcquireResponse = JSON.parse(text);
```

Comment:
nit: you should be able to do `await res.json()` directly
https://developer.mozilla.org/en-US/docs/Web/API/Response/json